test/functional/tests/security/fuzzy/kernel/fuzzy_with_io/fuzzy_io_class/test_fuzzy_io_class_config_priority.py

```diff
@@ -0,0 +1,104 @@
+    mount_point,
+)
+
+io_class_file_path = "/root/Fuzzy/ioclass.csv"
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.parametrizex("cleaning_policy", CleaningPolicy)
+@pytest.mark.parametrizex("unaligned_io", UnalignedIo)
+@pytest.mark.parametrizex("use_io_scheduler", UseIoScheduler)
+def test_fuzzy_io_class_config_priority(
+    cache_mode, cache_line_size, cleaning_policy, unaligned_io, use_io_scheduler
+):
+    """
+    title: Fuzzy test for IO class configuration content – IO class priority.
+    description: Using Peach Fuzzer check Open CAS ability of handling wrong IO class priority
+        in IO class configuration file.
+    pass_criteria:
+        - System did not crash
+        - Open CAS still works.
+    """
+    with TestRun.step(
+        "Start cache with configuration and add core device, make filesystem and " "mount it"
+    ):
+        cache_disk = TestRun.disks["cache"]
+        core_disk = TestRun.disks["core"]
+        cache, core = prepare_cas_instance(
+            cache_device=cache_disk,
+            core_device=core_disk,
+            cache_mode=cache_mode,
+            cache_line_size=cache_line_size,
+            kernel_params=KernelParameters(unaligned_io, use_io_scheduler),
+            cleaning_policy=cleaning_policy,
+            mount_point=mount_point,
+        )
+    with TestRun.step("Run fio in background"):
+        fio = get_basic_workload(mount_point)
+        fio_pid = fio.run_in_background()
+        io_class = IoClass(class_id=1, rule=f"file_size:{Operator.le.name}:97517568", priority=255)
+
+    with TestRun.step("Prepare PeachFuzzer"):
+        PeachFuzzer.generate_config(get_fuzz_config("uint.yml"))
+        parameters = PeachFuzzer.generate_peach_fuzzer_parameters(TestRun.usr.fuzzy_iter_count)
+
+    for index, parameter in TestRun.iteration(
+        enumerate(parameters), f"Run command {TestRun.usr.fuzzy_iter_count} times"
+    ):
+        with TestRun.step(f"Iteration {index + 1}"):
+            parameter = parameter.decode("ascii", "ignore")
+            io_class.priority = parameter
+            IoClass.save_list_to_config_file(
+                ioclass_list=[io_class],
+                add_default_rule=False,
+                ioclass_config_path=io_class_file_path,
+            )
+            cmd = get_cmd(
+                command=load_io_classes_cmd(str(core.cache_id), io_class_file_path),
+                param=parameter,
+            )
+            run_cmd_and_validate(cmd=cmd, value_name="Priority", is_valid=__is_valid(parameter))
+
+    with TestRun.step("Stop 'fio'"):
+        TestRun.executor.kill_process(fio_pid)
+
+
+def __is_valid(parameter):
+    if parameter == "":
+        return True
+    elif parameter.isdigit() and -1 <= int(parameter) <= 255:
```

Comment:
hmm we cannot use isdigit() for checks :( in previous tests as well :(  
```
>>> "3³3".isdigit()
True
>>> "-1".isdigit()
False
```

Reply:
also int("3³3") throws exception

Reply:
isdecimal() takes care of the exponents, but still doesn't handle negative values